forms-flow-components/src/components/index.ts

```diff
@@ -1 +1,2 @@
-export * from "./SvgIcons";
\ No newline at end of file
+export * from "./SvgIcons";
+export { default as CustomButton } from "./CustomComponents/Button";
```

Comment:
replace this line with :
export * from "./CustomComponents/Button";

Reply:
updated